doc/user/content/installation/appendix-production-guide.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Production guide"
+description: "Production guide/checklist for Self-Managed Materialize."
+draft: true
```

Comment:
This is just a draft.  Figured as we update our DR doc, we can also refashion key points as a checklist for production.